docs/content/config/advanced/kubernetes.md

```diff
@@ -448,8 +448,8 @@ With this approach,
 
 - it is not possible to access the mailserver via cluster-DNS, as the PROXY protocol is required for incoming connections.
 
-[Kustomize]: https://kustomize.io/
-[TLS]: ../security/ssl.md
+[docs-kustomize]: https://kustomize.io/
```

Comment:
I think there was a misunderstanding here, `kustomize` is an external link, not internal docs.

Usually that distinction is clear as I placed all the internal `docs-` links above the others. For example below is `dockerhub-haproxy`. You can continue to use `kustomize`, but `docs-` is misleading here :sweat_smile:

Reply:
I will change this, but I need to adjust the link above so I cannot directly commit your suggested changes.